src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactory.cs

```diff
@@ -90,31 +93,51 @@ public static TagHelperDesignTimeDescriptor CreateDescriptor(Type type)
             return null;
         }
 
-        private static DocumentationDescriptor CreateDocumentationDescriptor(Assembly assembly, string id)
+        private XmlDocumentationProvider GetXmlDocumentationProvider(Assembly assembly)
         {
-            var assemblyLocation = assembly.Location;
+            var hashCodeCombiner = HashCodeCombiner.Start();
+            hashCodeCombiner.Add(assembly);
+            hashCodeCombiner.Add(CultureInfo.CurrentCulture);
```

Comment:
In design time scenarios, can `CurrentCulture` change half way? Could we get away with simply varying by the `Assembly`?